.travis.yml

```diff
@@ -116,6 +116,7 @@ install:
       brew update > /dev/null ;
       brew install ccache ;
       brew install gawk ;
+      brew upgrade gnupg ;
```

Comment:
@jandom can you explain this change?

Reply:
@GiovanniBussi good catch! that was just to try the MacOS build to work – it complains about a gnupg version being missing (while another version for the package is present). I was just shooting in the dark here, clearly didn't make a difference

This has been removed now

Reply:
Sorry misspoke, now its removed